tokio/src/runtime/scheduler/multi_thread/queue.rs

```diff
@@ -10,6 +9,20 @@ use std::mem::MaybeUninit;
 use std::ptr;
 use std::sync::atomic::Ordering::{AcqRel, Acquire, Relaxed, Release};
 
+// Use wider integers when possible to increase ABA resilience.
+cfg_has_atomic_u64! {
+    pub(crate) type UnsignedShort = u32;
+    pub(crate) type UnsignedLong = u64;
+    pub(crate) type AtomicUnsignedShort = crate::loom::sync::atomic::AtomicU32;
+    pub(crate) type AtomicUnsignedLong = crate::loom::sync::atomic::AtomicU64;
+}
+cfg_not_has_atomic_u64! {
+    pub(crate) type UnsignedShort = u16;
+    pub(crate) type UnsignedLong = u32;
+    pub(crate) type AtomicUnsignedShort = crate::loom::sync::atomic::AtomicU16;
+    pub(crate) type AtomicUnsignedLong = crate::loom::sync::atomic::AtomicU32;
+}
```

Comment:
Seems like these don't need `pub(crate)`.

Reply:
oops, sorry, a bad case of copy-pasta...